sdk/state/integration/helpers_test.go

```diff
@@ -174,11 +272,58 @@ func retry(maxAttempts int, f func() error) (err error) {
 		if err == nil {
 			return
 		}
+		time.Sleep(2 * time.Second)
```

Comment:
can we make this a constant at the top of the file so we can easily modify and not have to search for this number `2`?

Reply:
taking a look again it actually looks like just using the retry without the sleep works, so I removed it 👍 
https://github.com/stellar/experimental-payment-channels/pull/86/commits/02bf876b0f155e362f1c09ec93fed785302dd03e